### PR TITLE
(GH-66) added missing tags

### LINKED ISSUE #66: Cake.ExtendedNuGet: Add cake-addin tag to NuGet package
The [NuGet Gallery](https://nuget.org) will soon include a "Cake" tab with instructions on how to use NuGet packages in Cake build scripts. I'd like to suggest that you include the tag `cake-addin` in the NuGet package of this addin, so that the NuGet Gallery can display the correct instructions to install this addin.

The UI will look similar to this:

image

Documentation on best practices on tags for Cake addins: https://cakebuild.net/docs/extending/addins/best-practices#tags

### PR DESCRIPTION
**Pull request by nils-a**
fixes #66

### CODE REVIEW

File: source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <Title>Cake.ExtendedNuGet</Title>
     <Description>Cake Build addon to provide Aliases for NuGet package inspection.</Description>
     <Summary>Cake build aliases for NuGet package inspection.</Summary>
-    <PackageTags>Cake Script Build</PackageTags>
+    <PackageTags>cake, script, build, cake-build, addin, cake-contrib, cake-addin, nuget</PackageTags>
```
Comment (augustoproiete) [inline source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj:26]:
```suggestion
    <PackageTags>cake;script;build;cake-build;addin;cake-contrib;cake-addin;nuget</PackageTags>
```

[Tags must be either space or semicolon-delimited](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#important-nuget-package-metadata)

**Review by augustoproiete — CHANGES_REQUESTED** [review]
Almost there

File: source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj (hunk 1 above)
Comment (nils-a) [reply source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj:26]: Does it matter? I feel I have "more" done using the ", " as delimiter 😲

Comment (nils-a) [reply source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj:26]: Hm. I guess NuGet gallery is prepared for someone like me: Using ", " works, despite the specs.
I wonder whether we should check that in CakeContrib.Guidelines, too.

Comment (augustoproiete) [reply source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj]: I suspect using `,` as separator might work (undocumented) or maybe it'll put everything as one giant tag :smile:

Some interesting comments here: https://github.com/dotnet/docs/issues/13553#issuecomment-518444966

**Review by augustoproiete — APPROVED** [review]
LGTM

**Pull request merged**